Process Participant Factory (#11)
rticommunity/rticonnextdds-xml-output-utility: README.md

| qosFile | Absolute or relative path of the QoS XML configuration file you want to analyze. You can also specify multiple files by separating them with a semicolon ';' and surrounding them with single quotes. For example: ``` 'file_path_1;file_path_2;file_path_3' ``` | OPTIONAL: The standard QoS XML files as defined in the User's manual will still be loaded |
| outputFile | Filename where the utility will output the QoS XML values | OPTIONAL: If not specified the output will be to the console |
| qosProfile | Fully qualified path of a QoS Profile. For example: QoSLibraryName::QoSProfileName | OPTIONAL: The ``` <qos_profile>``` with ``` is_default_qos="true" ``` will be selected OR the default values will be returned for the -qosTag |

> | qosTag | XML tag name who QoS values you want to be fetched. You can also select a subtag by separating it with a '/'. For example: ``` datawriter_qos/history ``` or ``` domain_participant_qos/property ``` | REQUIRED: Allowed values = {datawriter_qos, datareader_qos, topic_qos, domain_participant_qos (participant_qos is being deprecated), domain_participant_factory_qos, publisher_qos, subscriber_qos} |

Reviewer: ```suggestion
| qosTag | XML tag name of the QoS values you want to be fetched. You can also select a subtag by separating it with a '/'. For example: ``` datawriter_qos/history ``` or ``` domain_participant_qos/property ``` | REQUIRED: Allowed values = {datawriter_qos, datareader_qos, topic_qos, domain_participant_qos (participant_qos is being deprecated), domain_participant_factory_qos, publisher_qos, subscriber_qos} |
```
